content/applications/sales/subscriptions/ecommerce.rst

```diff
@@ -0,0 +1,52 @@
+=======================================
+Use subscriptions in the eCommerce shop
+=======================================
+
+Subscription products can be sold in the Odoo *eCommerce* shop just like regular sales products.
+
+However, by default, the eCommerce product page only displays the shortest recurrence period
+listed in the :guilabel:`Time-based pricing` tab of the product form. For example, if a
+subscription product has *monthly* and *yearly* recurrence periods configured, then only the monthly
+price appears on the eCommerce page for that product by default.
```

Comment:
100th char breaks.
I shouldn't see these by the time they hit final review 😉 
cc: @samueljlieber 
```suggestion
However, by default, the eCommerce product page only displays the shortest recurrence period listed
in the :guilabel:`Time-based pricing` tab of the product form. For example, if a subscription
product has *monthly* and *yearly* recurrence periods configured, then only the monthly price
appears on the eCommerce page for that product by default.
```